Config/DefaultEngine.ini

```diff
@@ -220,8 +220,8 @@ bUseManualIPAddress=False
 ManualIPAddress=
 
 [/Script/IOSRuntimeSettings.IOSRuntimeSettings]
-MobileProvision=fc364b1f-31f5-43e1-8104-8985132f4e6f.mobileprovision
-SigningCertificate=Apple Development: Nisarg Desai (GF42H6D58V)
+MobileProvision=58712ff3-db48-40b0-ad2c-f08f85eb2557.mobileprovision
+SigningCertificate=iPhone Distribution: New Relic Inc (SU7SUNGZJP)
```

Comment:
Why is the New Relic team distribution required in this Repo?

Reply:
i think i need to delete this file.